### ISSUE TITLE
[Parser] Crash with escaped characters

### ISSUE THREAD

**Issue by Graham42** [thorbenvh8/prettier-java#54]
Sample code
```java
public class MyTest {
	final char singleQuote = '\'';
	final char backslash = '\\';
}
```

Stacktrace
```
[error] MyTest.java: Error: Sad sad panda, parsing errors detected in line: 2, column: 30!
[error] Expecting: one of these possible Token sequences:
[error]   1. [';']
[error]   2. [';']
[error] but found: '';
[error]         final char backslash = ''
[error]     at Object.parse (/home/grahamm/<snipped-project-dir>/node_modules/chevrotain-java/src/index.js:20:11)
[error]     at Object.parse (/home/grahamm/<snipped-project-dir>/node_modules/prettier-plugin-java/src/parser.js:6:25)
[error]     at Object.parse$3 [as parse] (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:23712:19)
[error]     at formatWithCursor (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:29396:27)
[error]     at Object.formatWithCursor (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:29697:12)
[error]     at format$1 (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:35857:21)
[error]     at eachFilename (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:36082:16)
[error]     at filePaths.forEach.filePath (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:36022:7)
[error]     at Array.forEach (<anonymous>)
[error]     at eachFilename (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:36021:15)
[error]     at Object.formatFiles (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:36038:3)
[error]     at Object.run (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:36663:14)
[error]     at Object.<anonymous> (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:36678:5)
[error]     at Module._compile (module.js:652:30)
[error]     at Object.Module._extensions..js (module.js:663:10)
[error]     at Module.load (module.js:565:32)
```

**Comment by thorbenvh8** [comment]
Thanks, fixed in next release!

**Comment by thorbenvh8** [comment]
Fixed in 0.0.10!
Thanks for the raising the issue! 👍
